students/Craig_Morton/lesson09/mailroom_data.py

```diff
@@ -0,0 +1,42 @@
+# ------------------------------------------------- #
+# Title: Lesson 9, pt 3/4 mail room 5 data
+# Dev:   Craig Morton
+# Date:  9/26/2018
+# Change Log: CraigM, 9/26/2018, mail room 5 data
+# ------------------------------------------------- #
+
+
+class Donor:
+    def __init__(self, firstname, lastname):
+        self.name = (firstname, lastname)
+        self.donations = list()
+
+    def add_donation(self, amount):
+        self.donations.append(amount)
+
+    def get_donations(self):
+        return self.donations
+
+    def get_key(self):
+        return self.name
+
+    def get_name(self):
+        return "{0} {1}".format(*self.name)
```

Comment:
above class should also have attributes for average, total, and sum donations values.